lib/utils/validators.js

```diff
@@ -0,0 +1,42 @@
+class Validator {
+  constructor(test, errorMessage) {
+    this.test = test || false;
```

Comment:
Instead of calling this `test` which can also be a verb, how about we call it `pattern`?  That way it also indicates that its value should be a regular expression.

Also, I think it would be good at some point to have a single Validator support multiple patterns and will pass validation if one of the patterns match.  Specifically I'm thinking about phone numbers.